remoteworkitem/remoteworkitem.go

```diff
@@ -117,11 +120,32 @@ func (jira JiraRemoteWorkItem) Get(field AttributeExpression) interface{} {
 	return jira.issue[string(field)]
 }
 
+var issueStateMaps = map[string]string{
+	"open":   "open",
+	"closed": "closed",
+	"merged": "resolved",
+}
+
+//Mapping of workitem state
+func mapIssueStates(WorkItem app.WorkItem, issuemap IssueStateMap) (app.WorkItem, error) {
+	var currentstate string
+	for _, _ = range WorkItem.Fields {
+		for from, to := range issuemap {
+			if from == WorkItem.Fields["system.state"] {
+				currentstate = to
+			}
+		}
+		WorkItem.Fields["system.state"] = currentstate
+	}
+	return WorkItem, nil
+}
+
 // Map maps the remote WorkItem to a local WorkItem
 func Map(item AttributeAccessor, mapping WorkItemMap) (app.WorkItem, error) {
```

Comment:
@Ritsyy 
In ` func Map(item AttributeAccessor, mapping WorkItemMap)` 
the `WorkItemMap` object should have the information of the mapping information of states.